ewmap.py

```diff
@@ -1289,7 +1307,121 @@ def retrieve_locked_districts(id_server):
 			)
 		))
 
+async def survey(cmd):
+	user_data = EwUser(member=cmd.message.author)
+	district_data = EwDistrict(district=user_data.poi, id_server=user_data.id_server)
+	poi = ewcfg.id_to_poi.get(user_data.poi)
+
+	# get information about slime levels in the district
+	slimes = district_data.slimes
+	slimes_resp = ""
+	if slimes < 10000:
+		slimes_resp += "There are a few specks of slime splattered across the city streets."
+	elif slimes < 100000:
+		slimes_resp += "There are sparse puddles of slime filling potholes in the cracked city streets."
+	elif slimes < 1000000:
+		slimes_resp += "There are good amounts of slime pooling around storm drains and craters in the rundown city streets."
+	else:
+		slimes_resp += "There are large heaps of slime shoveled into piles to clear the way for cars and pedestrians on the slime-soaked city streets."
```

Comment:
at this point we should probably make some functions that generate these responses for all commands that need them